src/main/java/io/airbyte/testingtool/scenario/action/connection/ActionCreateConnectionCustom.java

```diff
@@ -34,6 +34,7 @@ protected ConnectionCreate getConnectionCreateConfig() throws ApiException {
     var requestBody = new SourceDiscoverSchemaRequestBody();
     requestBody.setSourceId(sourceInstance.getId());
     requestBody.setDisableCache(true);
+    updateUseNormalizationFlag(Objects.nonNull(getSettings().getNormalization()) ? getSettings().getNormalization() : false);
```

Comment:
This line will be removed after changes from the comments above.

Reply:
done, thanks for the comment